umbraco-workflow/notifications.md

```diff
@@ -0,0 +1,769 @@
+# Using Notifications
+
+Umbraco Workflow uses Notifications to allow you to hook into the processes for the backoffice. For example, you might want to execute some code every time an approval group is updated. Notifications allow you to do that. For more information on how Umbraco implements Notifications, see the [Using Notifications](https://docs.umbraco.com/umbraco-cms/reference/notifications) article.
+
+## Notifications
+
+All notifications reside in the `Umbraco.Workflow.Core.Notifications` namespace and are postfixed with `Notification`.
+
+Typically, available notifications exist in pairs, with a "before" and "after" notification. For example, the `SettingsService` class publishes both a `WorkflowSettingsSavingNotification` and a `WorkflowSettingsSavedNotification` when settings are modified.
+
+Which one you want to use depends on what you want to achieve. If you want to cancel the action, you will use the "before" notification and use the CancelOperation method on the notification to cancel it. If you want to execute some code after the settings have been updated, then you would use the "after" notification.
+
+## ContentReviewService Notifications
+
+<table>
+  <tr>
+    <th>Notification</th>
+    <th>Members</th>
+    <th>Description</th>
+  </tr>
+
+  <tr>
+    <td>WorkflowContentReviewsConfigSavingNotification</td>
+    <td>
+      <ul>
+        <li>IEnumerable<IWorkflowConfig> SavedEntities</li>
+        <li>EventMessages Messages</li>
+        <li>IDictionary&ltstring,object&gt State</li>
+        <li>bool Cancel</li>
```

Comment:
Hey @eshanrnh we're getting value warnings for this line, but it's correct for `bool` to be lowercase, so will leave as-is. It's only a warning after all 😄

Reply:
Yup, I agree 😀 That's why I ignored that warning 😉